nixos/modules/services/web-apps/dokuwiki.nix

```diff
@@ -75,9 +153,22 @@ let
         };
 
         acl = mkOption {
-          type = types.nullOr types.lines;
+          type = with types; nullOr (oneOf [ lines (listOf (submodule aclOpts)) ]);
```

Comment:
What are the opinions on entirely deprecating support for plaintext ACLs? Since their definition is very straight forward (lines of three, space or tab separated, values) I am confident that `aclOpts` does not impose any limits over the big string.

We could add a warning for everyone passing strings and remove that in time for 23.05.

Reply:
I'm not using any acls so I cannot tell but sounds fine to me